pallets/phala/src/mock.rs

```diff
@@ -130,14 +131,17 @@ impl mining::Config for Test {
 	type OnReclaim = PhalaStakePool;
 	type OnStopped = PhalaStakePool;
 	type OnTreasurySettled = ();
+	type UpdateTokenomicOrigin = frame_system::EnsureRoot<u64>;
 }
 
 impl stakepool::Config for Test {
 	type Event = Event;
 	type Currency = Balances;
 	type MinContribution = MinContribution;
 	type GracePeriod = MiningGracePeriod;
+	type MiningEnabledDefault = MiningEnabledDefault;
 	type OnSlashed = ();
+	type MiningSwitchOrigin = frame_system::EnsureRoot<u64>;
```

Comment:
`frame_system::EnsureRoot<Self::AccountId>` is better

Reply:
Done